amundsen_application/static/js/components/DashboardPage/ImagePreview/index.spec.tsx

```diff
@@ -71,29 +72,80 @@ describe('ImagePreview', () => {
         });
       });
 
-      describe('when not loading', () => {
+      describe('when loaded', () => {
         let props;
         let wrapper;
+
         beforeAll(() => {
           const setupResult = setup();
           props = setupResult.props
           wrapper = setupResult.wrapper;
-          wrapper.instance().setState({ isLoading: false, hasError:false });
+          wrapper.instance().setState({ isLoading: false, hasError: false });
         });
+
         it('renders visible img with correct props', () => {
           const elementProps = wrapper.find('img').props();
-          expect(elementProps.style).toEqual({ visibility: 'visible' });
```

Comment:
Is this redundant in some way? I find it valuable to verify any logic that affects what shows up on the page.

Reply:
It started breaking when I switched to mount. 
I didn't spend too much time trying to debug it though, I could do it now if you want.

Reply:
Ah, I only needed the `wrapper.update()`, I will bring it back!